extensions/amp-fit-text/amp-fit-text.md

```diff
@@ -108,4 +118,4 @@ You can style the `amp-fit-text` with standard CSS. In particular, you can use `
 
 ## Validation
 
-See [amp-fit-text rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-fit-text/validator-amp-fit-text.protoascii) in the AMP validator specification.
+Please note that version `1.0` of `amp-fit-text` is not currently valid AMP, though this is an imminent milestone in the Bento roadmap. Use version `0.1` for a similar experience on a valid AMP document.
```

Comment:
At this point I'm wondering if we need this section or should we just imply this since this version is marked as `experimental`?

Reply:
Good point, removed.